runtime/compiler/control/CompilationThread.cpp

```diff
@@ -13368,13 +13367,16 @@ TR::CompilationInfo::methodCanBeRemotelyCompiled(const char *methodSig, TR::Meth
    }
 
 void
-TR::CompilationInfo::notifyCompilationThreadsOfDeserializerReset()
+TR::CompilationInfo::notifyThreadsOfDeserializerReset()
```

Comment:
Since the method only notifies the compilation threads, the old name of the method was more appropriate: `notifyCompilationThreadsOfDeserializerReset`

Reply:
Ah yeah; I think I had originally anticipated using this to notify all other threads, but it ended up not working out that way. Updated in force push.